test/sql/add_files/add_file_footer_size.test

```diff
@@ -0,0 +1,25 @@
+# name: test/sql/add_files/add_file_footer_size.test
+# description: test if ducklake adds file footer size
+# group: [add_files]
+
+require ducklake
+
+require parquet
+
+statement ok
+ATTACH 'ducklake:__TEST_DIR__/add_file_footer_size.db' AS ducklake (DATA_PATH '__TEST_DIR__/ducklake_empty_files/', METADATA_CATALOG 'metadata');
+
+statement ok
+CREATE TABLE ducklake.test(col1 INTEGER, col2 VARCHAR);
+
+# write a parquet file outside of DuckLake
+statement ok
+COPY (SELECT 200 col1, 'world' col2) TO '__TEST_DIR__/data.parquet';
+
+statement ok
+CALL ducklake_add_data_files('ducklake', 'test', '__TEST_DIR__/data.parquet')
+
+query I
+SELECT data_file_footer_size FROM ducklake_list_files('ducklake', 'test')
```

Comment:
Let's just do `footer_size > 0`, otherwise this can randomly break (as already evidenced by the upstream change)